src/plugins/home/public/application/components/solutions_section/solution_panel.tsx

```diff
@@ -64,9 +64,10 @@ interface Props {
   addBasePath: (path: string) => string;
   solution: FeatureCatalogueSolution;
   apps?: FeatureCatalogueEntry[];
+  branding: Branding;
```

Comment:
nit can this be part of the `solution`

Reply:
After diving into solution, it is not suitable to include branding within solution, since solution is being used at other places in the application where we do not need custom branding.